libafl/src/stages/mod.rs

```diff
@@ -199,7 +201,7 @@ where
                 state.set_current_stage_idx(StageId(Self::LEN))?;
 
                 let stage = &mut self.0;
-                stage.perform_restartable(fuzzer, executor, state, manager)?;
+                stage.perform(fuzzer, executor, state, manager)?;
```

Comment:
this is wrong, we should always call `perform_restartable`

Reply:
why?
this is not from resuming

Reply:
because if it is after resuming then it goes to the above branch